includes/class-wc-payments-account.php

```diff
@@ -419,7 +419,15 @@ private function get_login_url() {
 	 * @return string Stripe account login url.
 	 */
 	public static function get_connect_url() {
-		return wp_nonce_url( add_query_arg( [ 'wcpay-connect' => '1' ] ), 'wcpay-connect' );
+		$connect_url = add_query_arg(
```

Comment:
Should we remove this var `$connect_url` too as we're not using it anymore?

Reply:
Definitely 👍🏻